code/modules/admin/admin_verbs.dm

```diff
@@ -1104,6 +1105,68 @@ var/list/admin_verbs_mod = list(
 	log_admin("[key_name(src)] sent a fax to all machines.: [sent]")
 	message_admins("[key_name_admin(src)] sent a fax to all machines.", 1)
 
+/client/proc/create_map_element()
+	set category = "Admin"
+	set name = "Load Map Element"
+	set desc = "Loads a map element - a vault, an away mission or something else."
+
+	if(!check_rights(R_SPAWN))
+		return
+
+	var/datum/map_element/ME
+	var/mission_to_load = alert(usr, "How do you want to select the map element?", "Map element loading", "Choose a /datum/map_element object", "Load external .dmm file", "Cancel")
+	switch(mission_to_load)
+		if("Choose a /datum/map_element object")
+			var/new_map_element = input(usr, "Please select the map element object.", "Map element loading") as null|anything in typesof(/datum/map_element) - /datum/map_element
+			if(!new_map_element) return
+
+			ME = new new_map_element
+			log_admin("[key_name(src)] is trying to load [ME.file_path].")
+
+		if("Load external .dmm file")
+			to_chat(src, "<span class='danger'>Do not load very large maps or files that aren't BYOND maps. If you want to be sure that your map won't hang up the game, try loading it on a local server first.</span>")
+			ME = new /datum/map_element
+			log_admin("[key_name(src)] is trying to load an external map file.")
+			var/new_file_path = input(usr, "Select a .dmm file.    WARNING: Very large map files WILL crash the server. Loading them is punishable by death.", "Map element loading") as null|file
+			if(!new_file_path) return
+
+			log_admin("[key_name(src)] has selected [new_file_path] for loading.")
+			ME.file_path = new_file_path
+		else
+			return
+
+	var/x_coord
+	var/y_coord
+	var/z_coord
+
+	switch(alert(usr, "Select a location for the new map element", "Map element loading", "Use my current location", "Input coordinates", "Cancel"))
+		if("Use my current location")
+			var/turf/new_location = get_turf(usr)
+			if(!new_location)
+				return
+
+			x_coord = new_location.x
+			y_coord = new_location.y
+			z_coord = new_location.z
+
+		if("Input coordinates")
+			x_coord = input(usr, "Input the X coordinate: ", "Map element loading") as null|num
+			if(x_coord == null) return
+
+			y_coord = input(usr, "Input the Y coordinate (X = [x_coord]): ", "Map element loading") as null|num
+			if(y_coord == null) return
+
+			z_coord = input(usr, "Input the Z coordinate. If it's higher than [world.maxz], a new Z-level will be created (X = [x_coord], Y = [y_coord]): ", "Map element loading") as null|num
+			if(z_coord == null) return
+
+		if("Cancel")
+			return
+
+	log_admin("[key_name(src)] is loading [ME.file_path] at [x_coord], [y_coord], [z_coord]")
+	message_admins("[key_name_admin(src)] is loading [ME.file_path] at [x_coord], [y_coord], [z_coord]")
+	ME.load(x_coord, y_coord, z_coord)
```

Comment:
My only question is shouldn't this include some kind of sanity/warning for when a loaded maps width/height are too large and will cause it to collide with the edge of the map or another area? 

It seems like that should be possible